mantidproject/mantid
# Update ISIS Reflectometry documentation

### Description of work

#### Summary of work

Updates screenshots, and a few bits of information and typos in the ISIS Reflectometry interface and technique documentation. There are a couple of sections that need input from our scientists to make sure they are updated accurately (see linked issue) so these will be done in a later PR.



Refs #38128. 




#### Further detail of work

- Updates screenshots that are now out of date (most due to quite minor changes in the interface).
- Fixes some typos.
- Adds a brief mention of the export feature for search results in the interface docs.
- Refers to the ability to output ORSO files in the technique docs.
- Adds algorithm references and corrects some table formatting issues.


### To test:


Build the `docs-html` target and check that the following files render OK:

- `techniques->ISIS_Reflectometry.html`
- `interfaces->reflectometry->ISIS Reflectometry.html`

*This does not require release notes* because **this is just a documentation update.**



---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
